code/modules/vehicles/tank/tank.dm

```diff
@@ -164,7 +164,39 @@
 	update_icon()
 
 	return TRUE
+/obj/vehicle/multitile/tank/MouseDrop_T(mob/M, mob/user)
+	. = ..()
+	if((M != user) || !isxeno(user))
+		return
 
+	if(health > 0)
+		to_chat(user, SPAN_XENO("We can't jump over [src] until it is destroyed!"))
+		return
+
+	var/turf/current_turf = get_turf(user)
+	var/dir_to_go = get_dir(current_turf, src)
+	for(var/i in 1 to 3)
```

Comment:
is this hardcoded to be the width of the vehicle

Reply:
yes